build.gradle

```diff
@@ -0,0 +1,65 @@
+apply plugin: 'java'
+apply plugin: 'application'
+apply plugin: 'idea'
+apply from: 'gradle/quality.gradle'
+
+sourceCompatibility = '1.7'
+targetCompatibility = '1.7'
+
+[compileJava, compileTestJava]*.options*.encoding = 'UTF-8'
+
+mainClassName = "org.apache.mesos.hdfs.Main"
+
+group       = "org.apache.mesos"
+version     = "0.0.2-SNAPSHOT"
+
+ext {
+    mesosVer = "0.21.1"
+    slf4jVersion = "1.7.10"
+    logback_classicVersion = "1.1.2"
+    hadoopVersion = "2.4.0"
+    jettyVersion= "9.2.2.v20140723"
+    joda_timeVersion = "2.4"
+    jmteVersion= "3.0"
+    guiceVersion = "3.0"
+    guavaVersion= "18.0"
+    junitVersion = "4.11"
+    mockitoVersion = "1.9.5"
+}
+
+repositories {
+	mavenLocal()
+	mavenCentral()
+}
+
+dependencies {
+	compile "org.apache.mesos:mesos:${mesosVer}"
+    
+    compile "org.slf4j:log4j-over-slf4j:${slf4jVersion}"
+	compile "org.slf4j:jcl-over-slf4j:${slf4jVersion}"	
+	
+	compile "ch.qos.logback:logback-classic:${logback_classicVersion}"
+
+	compile ("org.apache.hadoop:hadoop-common:${hadoopVersion}") {
```

Comment:
great question.  The gradle build configuration exactly mimics what was in the maven build pom.xm for dependencies.  Since we have an issue to track this, I will assume it is fine here.